lib/modules/platform/github/index.ts

```diff
@@ -127,7 +128,7 @@ export async function initPlatform({
 }: PlatformParams): Promise<PlatformResult> {
   let token = originalToken;
   if (!token) {
-    throw new Error('Init: You must configure a GitHub token');
+    throw new Error('Init: ' + gt.gettext('You must configure a GitHub token'));
```

Comment:
This would be part of admin logs and not part of end-user facing text

Reply:
OK, I translate this only for a quick test. I will roll it back.